src/middlewared/middlewared/plugins/truecommand/wireguard.py

```diff
@@ -126,7 +126,7 @@ async def start_truecommand_service(self):
                 config[k] for k in ('wg_private_key', 'remote_address', 'endpoint', 'tc_public_key', 'wg_address')
             ):
                 await self.middleware.call('service.start', 'truecommand')
```

Comment:
Wait, should we add `{"ha_propagate": false}` here too?

Reply:
The service itself is black listed, so that's redundant ?